rebar.config

```diff
@@ -61,6 +61,12 @@
   {short, [{erl_opts, [{d, 'EQC_SHORT_TEST'}]}]},
   {long, [{erl_opts, [{d, 'EQC_LONG_TEST'}]}]},
   {prod, [{relx, [{dev_mode, false}]}]},
+  {deb, 
+	  [{relx, 
+		  [{dev_mode, false},
+		  {overlay_vars, "rel/vars/deb.config"}]},
+	  {cuttlefish,
```

Comment:
Oh I see why, you change the file name. Sorry I disagree with doing that. Different configuration file names on different OS's is a total horrible, that is not OS specific.